### DISCUSSION TITLE
Can we save all used models in one nemo file ?

### DISCUSSION THREAD

**Discussion by viralrupapara36** [NVIDIA-NeMo/NeMo#5434] [Q&A]
I'm working on Speaker Diarization with ASR and there is titanet, vad, msdd & conformer-ctc models are used.
Instead of four can we save it as one model file and later on can we use that one file for testing ?

**Comment by titu1994** [comment]
@nithinraok / @tango4j

**Comment by tango4j** [comment]
For clustering diarizer, we can export everything to one instance. For ASR with diarization, we haven't worked on that yet.
